back/config/schemas/settings.schema.json.erb

```diff
@@ -1238,6 +1238,18 @@
         }
       },
 
+      "native_surveys_mapping_question": {
+        "type": "object",
+        "title": "Mapping question in in-platform surveys",
+        "description":  "Allow mapping question(s) in in-platform surveys. Currently limited to one type: Select a point on a map.",
```

Comment:
Just need to make sure that we change this description if and when the limitation changes.